src/js/components/DataTable/README.md

```diff
@@ -228,6 +440,18 @@ Whether to allow the user to resize column widths.
 boolean
 ```
 
+**rowProps**
+
+Row specific background, border, and pad, keyed by primary key value.
+      For example:
+      { "primary-key-value": { background: ..., border: ..., pad: ... }}.
+
+```
+{
```

Comment:
These braces look empty, was it the purpose?

Reply:
It was. I was trying to avoid repeating too much from the background, border, and pad properties on DataTable. I've added a little more description to try to help explain that.